work_queue_gantt.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python
+
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Author: Guanchao "Samuel" Huang
+# %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+# Packages and Definitions
+import re
+import sys
+import json
+
+import bokeh
```

Comment:
Wrap all the imports to bokeh in a `try:   except ImportError` block. On error, print the conda command to install bokeh.